src/main/java/database/helper/Settings.java

```diff
@@ -83,4 +76,27 @@ Path getFilePath(String directoryName) throws CreationDatabaseException {
             throw new CreationDatabaseException("Directory cannot be created: " + e.getMessage());
         }
     }
+
+    private void loadProperties() {
+        properties.clear();
```

Comment:
Consider creating new instance of `Properties` instead of cleaning. You can declare `private Properties properties` without creating object.